app/partials/register-form.cjsx

```diff
@@ -20,7 +20,7 @@ counterpart.registerTranslations 'en',
     whyUserName: 'You’ll use this name to log in. It will be shown publicly. '
     badChars: "Only letters, numbers, '.', '_', and '-'."
     nameConflict: 'That username is taken'
-    forgotPassword: 'Forget your password?'
+    forgotPassword: 'Forgot my password'
```

Comment:
```suggestion
    forgotPassword: 'Forgot your password'
```

Reply:
My bad. These keys should be stored in `/app/locales/en.js`. As they are here, they're not translated at all.